### PR TITLE
Updated Broker Legacy Manifest to use "Created" as source for "SentDate"

### PR DESCRIPTION
**Pull request by RagnarFatland-Avanade**
## Description
Updated Broker Legacy Manifest to use "Created" as source for "SentDate" and ensured this was correctly converted to a local DateTime without timezone info.

## Related Issue(s)
- #595 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [x] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

## Summary by CodeRabbit

- **New Features**
	- Updated the way `SentDate` is recorded, now reflecting the local time based on the entity's creation time.

- **Bug Fixes**
	- Enhanced tests to verify that the `SentDate` in the broker manifest matches the expected local date and time.

- **Tests**
	- Added assertions in existing test methods to ensure proper handling of date conversions and validate the accuracy of `SentDate`.

### CODE REVIEW

File: tests/Altinn.Broker.Tests/ManifestDownloadStreamTests.cs [hunk 1]
```diff
@@ -53,8 +56,30 @@ public async Task StreamWithManifest_AddManifest_ExistingManifestIsReplacedAsync
...
+        DateTime conv3 = (orgDateTimeOffset.ToLocalTime()).DateTime;
+
+        string s = "orgOffset: " + orgDateTimeOffset.ToString() + Environment.NewLine
+            + "orgDateTime: " + orgDateTime.ToString() + Environment.NewLine
+            + "conv1: " + conv1.ToString() + Environment.NewLine
+            + "conv2: " + conv2.ToString() + Environment.NewLine
+            + "conv2: " + conv3.ToString() + Environment.NewLine;
+
+        Assert.NotEmpty(s);
+    }
```
[the remark this answers is not in the document]
Comment (RagnarFatland-Avanade) [reply tests/Altinn.Broker.Tests/ManifestDownloadStreamTests.cs]: Agreed, I will remove this.

File: src/Altinn.Broker.Core/Helpers/BrokerServiceManifest.cs [hunk 2]
```diff
@@ -63,7 +63,7 @@ public static BrokerServiceManifest CreateManifest(this FileTransferEntity entit
             ExternalServiceEditionCode = resource.UseManifestFileShim == true ? resource.ExternalServiceEditionCodeLegacy : null,
             SendersReference = entity.SendersFileTransferReference,
             Reportee = entity.RecipientCurrentStatuses.First().Actor.ActorExternalId,
-            SentDate = DateTime.UtcNow,
+            SentDate = entity.Created.ToLocalTime().DateTime,
```
[the remark this answers is not in the document]
Comment (Andreass2) [reply src/Altinn.Broker.Core/Helpers/BrokerServiceManifest.cs:66]: Have you tested in environment or in a docker container?

[the remark this answers is not in the document]
Comment (RagnarFatland-Avanade) [reply src/Altinn.Broker.Core/Helpers/BrokerServiceManifest.cs:66]: UTC: This timestamp is used by the the Altinn 2 API Consumers, and in that sceario they are expecting the local time (without time offset).

An equivalent change was performed in Altinn 2 Codebase successfully in issue #599

**Review by Andreass2 — APPROVED** [review]

File: src/Altinn.Broker.Core/Helpers/BrokerServiceManifest.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (RagnarFatland-Avanade) [reply src/Altinn.Broker.Core/Helpers/BrokerServiceManifest.cs:66]: Conclusion: as our servers will be located in tha same timezone, there should not be any issues, and it is a tiny subset of our users who will be using this functionality. We accept the risk.

**Pull request merged**